blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_10.py

```diff
@@ -0,0 +1,53 @@
+"""Formula 5.10 from NEN-EN 1993-5:2008 Chapter 5 - Ultimate limit state."""
+
+from blueprints.codes.eurocode.nen_en_1993_5_2008 import NEN_EN_1993_5_2008
+from blueprints.codes.formula import Formula
+from blueprints.codes.latex_formula import LatexFormula
+from blueprints.type_alias import DIMENSIONLESS, KN
+from blueprints.validations import raise_if_less_or_equal_to_zero
+
+
+class Form5Dot10ReductionFactorShear(Formula):
+    """Class representing formula 5.10 for reduction factor for shear resistance of the cross-section."""
```

Comment:
![image](https://github.com/Blueprints-org/blueprints/assets/111974143/af5ea239-c0a7-46c2-aa0d-75c251fefd3b)

![image](https://github.com/Blueprints-org/blueprints/assets/111974143/992b6394-d5d0-431e-bc61-bb03fcb26fec)

```suggestion
    """Class representing formula 5.10 for reduction factor for shear area."""
```

Reply:
That sounds indeed like a better description. 👍